### PR TITLE
docs: updated navigation section

### PR DESCRIPTION
**Pull request by mehaknasir**
Add a description of your changes with links to any relevant material.

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:.

- [ ] Subject matter expert:
- [ ] Subject matter expert:
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support:
- [ ] PM review:

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ https://draftinf-496.readthedocs.io/en/latest/sfd_discussions/anatomy.html] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/open_edx_students/source/sfd_discussions/anatomy.rst [hunk 1]
```diff
@@ -106,98 +106,77 @@ course team, be sure to create your post as a question, so that the course
...
 
-Use the discussion navigation pane on the left side of the **Discussions**
-page to access discussion topics and posts in your course, including
-discussions that appear inline in course content.
+Use tabs on top of the Discussions page to access forum content in your course:
 
-You can view all discussion topics or only the posts that you are following,
-or you can view a specific topic. You can also filter posts by entering
-keywords or phrases that you want to find in post titles or text.
+ * All posts tab lists all the posts in the course.
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/anatomy.rst]: Please make the name of the tabs (All posts, My posts, Topics) bold everywhere.

File: en_us/open_edx_students/source/sfd_discussions/anatomy.rst [hunk 2]
```diff
@@ -106,98 +106,77 @@ course team, be sure to create your post as a question, so that the course
...
-       following. You can also filter and sort posts, and you are notified
-       about new unread responses and comments in posts.
+''''''''''''''
+All posts tab
+''''''''''''''
 
-The breadcrumb path above the navigation pane shows you the topic that you are
-currently viewing. In the following example, **Posts I'm Following** is
-selected.
+All posts show the list of all posts in the forum for this course that are visible to your cohort (if forums are divided by cohorts).
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/anatomy.rst]: Replace the entire line with "The All posts tab shows the list of all posts in the forum for this course."
Note that this change removes reference to cohorts.

File: en_us/open_edx_students/source/sfd_discussions/anatomy.rst [hunk 3]
```diff
@@ -106,98 +106,77 @@ course team, be sure to create your post as a question, so that the course
...
+7. A “Pinned” icon shows that a post has been pinned by moderators so that it always stays at the top of the topic list regardless of sort order.
 
-*  The post type icon indicates whether a post is a question or a discussion.
-   For more information, see :ref:`Determine Post Type`.
+8. A filled star button indicates that you are following the post. You can click on the icon to follow/unfollow the post. By default, you’ll be following the posts that you have created.
 
-*  The check mark icon indicates a "correct answer". The check mark icon
-   indicates that one of the responses to the post correctly answers the
-   question asked in the post.
+9. A filled like button indicates that you have liked the post. You can click the like to like and unlike a post. The count next to the like button indicates how many likes does the post have.
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/anatomy.rst]: add quotes on like --> "like"

Change "You can click the like to like" to "You can click the like button to like"

File: en_us/open_edx_students/source/sfd_discussions/anatomy.rst [hunk 4]
```diff
@@ -106,98 +106,77 @@ course team, be sure to create your post as a question, so that the course
...
-   not read.
+6. The number of new responses or comments that were added after you previously read a post is displayed next to the response count indicator. For more information, see :ref:`Keep Up with New Activity`.
 
-*  Posts that appear in gray type without a bar indicate posts that you have
-   read.
+7. A “Pinned” icon shows that a post has been pinned by moderators so that it always stays at the top of the topic list regardless of sort order.
 
-*  The post type icon indicates whether a post is a question or a discussion.
-   For more information, see :ref:`Determine Post Type`.
+8. A filled star button indicates that you are following the post. You can click on the icon to follow/unfollow the post. By default, you’ll be following the posts that you have created.
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/anatomy.rst]: add quotes on star --> "star".
Replace icon with button.

File: en_us/open_edx_students/source/sfd_discussions/anatomy.rst [hunk 5]
```diff
@@ -106,98 +106,77 @@ course team, be sure to create your post as a question, so that the course
...
-*  The check mark icon indicates a "correct answer". The check mark icon
-   indicates that one of the responses to the post correctly answers the
-   question asked in the post.
+9. A filled like button indicates that you have liked the post. You can click the like to like and unlike a post. The count next to the like button indicates how many likes does the post have.
 
-*  The response or comment count indicator shows the number of responses and
-   comments for a post. The number of new responses or comments that were
-   added after you previously read a post is displayed next to the indicator.
-   For more information, see :ref:`Keep Up with New Activity`.
+10. A “Staff” and “TA” label marks posts that were added by course staff and community teaching assistants (TAs), respectively.
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/anatomy.rst]: Replace with: "The “Staff” or “TA” label next to the username indicates that the user belongs to the course team or has a community teaching assistant (TA) role, respectively."

File: en_us/open_edx_students/source/sfd_discussions/anatomy.rst [hunk 6]
```diff
@@ -106,98 +106,77 @@ course team, be sure to create your post as a question, so that the course
...
 
-*  A "Pinned" label shows that a post has been pinned by administrators
-   so that it always stays at the top of the topic list regardless of sort
-   order.
+11. “Answered” label on summary of question and indicates that a response to this question has been marked as “answer” by a moderator.
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/anatomy.rst]: Replace with: The “Answered” label on summary of a question and indicates that a response to this question has been marked as an answer by a user having moderation privileges.

File: en_us/open_edx_students/source/sfd_discussions/anatomy.rst [hunk 7]
```diff
@@ -106,98 +106,77 @@ course team, be sure to create your post as a question, so that the course
...
-page to access discussion topics and posts in your course, including
-discussions that appear inline in course content.
+Use tabs on top of the Discussions page to access forum content in your course:
 
-You can view all discussion topics or only the posts that you are following,
-or you can view a specific topic. You can also filter posts by entering
-keywords or phrases that you want to find in post titles or text.
+ * All posts tab lists all the posts in the course.
+ * My posts tab lists all the posts that you have created.
+ * Topics tab lists all discussion topics in the course. Course-wide discussion topics appear on top of the list followed by content-wide topics, in the order that they exist in the course content. You can click on each topic to view the posts.
```
Comment (ayub02) [inline en_us/open_edx_students/source/sfd_discussions/anatomy.rst]: Reduce to: "The Topics tab lists all discussion topics in the course."

File: en_us/open_edx_students/source/sfd_discussions/anatomy.rst (hunk 1 above)
Comment (mehaknasir) [reply en_us/open_edx_students/source/sfd_discussions/anatomy.rst]: okay.

**Review by ayub02 — APPROVED** [review]

**Pull request merged**